lib/tasks/tenants.rake

```diff
@@ -0,0 +1,61 @@
+namespace :hyku do
```

Comment:
Is there a difference between this and the index.rake file?

Reply:
oops, i was doing this way too fast. I just updated it!